src/modules/controllers.ts

```diff
@@ -268,6 +274,45 @@ class ControllerModule extends VuexModule {
     return [];
   }
 
+  get userCustomTokens(): SolanaToken[] {
+    if (this.customTokens) {
+      return this.customTokens
+        .reduce((acc: SolanaToken[], current: TokenInfo) => {
+          const userToken = this.userTokens.find((token) => token.mintAddress === current.address);
+
+          if (userToken?.balance) {
```

Comment:
if userToken.balance === 0 , we should not add new customToken to the display
else we will see 2 token with same address and balance 0 ?

Reply:
yes the below line ignores if the user has balance for that token